molecule/command/test.py

```diff
@@ -110,8 +110,8 @@ def execute(self):
           'Molecule run (always).'))
 def test(ctx, scenario_name, driver_name, __all, destroy):  # pragma: no cover
     """
-    Test (lint, destroy, dependency, syntax, create, prepare, converge,
-          idempotence, side_effect, verify, destroy).
+    Test (lint, cleanup, destroy, dependency, syntax, create, prepare,
```

Comment:
lint, cleanup, destroy, dependency, syntax, create, prepare, converge, idempotence, side_effect, verify, cleanup, destroy.

Reply:
> Thanks for this fix! LGTM +1
> 
> As far as I understand, the `cleanup` -> `destroy` steps are in the sequence already but because there is a failure which stops execution, we have to manually call the `cleanup` -> `destroy` steps ourselves? Am I on the right track?

Yes, that is exactly what was happening. Because lint was directly calling destroy on errors, and not trying to follow the sequence. The better design I feel would be to use the sequence in some way, but that would require a lot more work.

Reply:
I think that this is good for now because it uses a similar pattern to the rest of molecule. Utilizing the sequence would indeed be a lot more work and would make this section different than how the rest of the molecule commands work internally.

Reply:
Agreed :+1: